docs/RELEASING.md

```diff
@@ -27,6 +27,20 @@ After the Elastic unified release is complete
 
 - Update the **BUILD** version ([example PR](https://github.com/elastic/connectors-python/pull/122)). Note that the Connectors project does not immediately bump to the next **PATCH** version. That won't happen until that patch release's FF date.
 
+## Releasing docker images
+
+To release the docker image, follow these steps:
+
+1. Make sure you're on the right tagged release commit, e.g. `v8.10.2.0`
+   - If no release tag exists yet, use the release branch, e.g. `8.10`
+2. Make sure the version in [VERSION](../connectors/VERSION) is correct
+3. Run `docker login -u <username> -p <password> docker.elastic.co` with credentials that allow for release
+4. Edit the [Makefile](../Makefile) to remove `-SNAPSHOT` from the `docker-build`, `docker-run` and `docker-push` steps
```

Comment:
Should we add a `docker-release` target?

Reply:
@wangch079 we aren't expecting to need to run this manually because it's automated, so I thought some instructions would be enough rather than a new makefile target.

Reply:
Is there a ticket for automation already?

Reply:
@sphilipse do you know if automating this in 8.11 has a ticket?